localnet/test_fanout.ts

```diff
@@ -0,0 +1,133 @@
+// NOTE(will): just getting this stuff out of the way for now
```

Comment:
would prefer if this remained uncommitted rather than commented out, in the future